.github/workflows/release.yml

```diff
@@ -44,18 +53,11 @@ jobs:
           make README.rst
 
       - name: Build and publish to PyPI Test
-        env:
-          TWINE_USERNAME: __token__
-          TWINE_PASSWORD: ${{ secrets.TEST_PYPI_TOKEN }}
-          TWINE_REPOSITORY_URL: "https://test.pypi.org/legacy/"
         run: |
-          sudo apt-get install -y twine
-          make pypi
+          poetry config repositories.testpypi https://test.pypi.org/legacy/
+          poetry publish --build -r testpypi -u __token__ -p ${{ secrets.TEST_PYPI_TOKEN }}
```

Comment:
Is there a way for us to set the PyPI and TestPyPI passwords on the environment instead of on the command-line?

Here is the problem - the commands executed will probably show up in the Github Actions execution history, and that might expose the password. 🤔  Safer to set it to an environment variable that will be then used directly by poetry.

Reply:
Turns out Github handles that automatically and masks the password, which is awesome:
<img width="805" alt="image" src="https://user-images.githubusercontent.com/3287067/215781805-98a43c95-688a-44b0-93f8-7b5576246cf6.png">

So no change needed here.